app/routes.py

```
@@ -4,10 +4,10 @@
 import uuid
 from typing import Any, Dict, List, Optional, Tuple
 from datetime import datetime
+from io import BytesIO
```

Ruff (0.8.2)

`io.BytesIO` imported but unused

Remove unused import: `io.BytesIO`

(F401)